plugins/scaffolder-backend/src/scaffolder/actions/builtin/fetch/cookiecutter.ts

```diff
@@ -37,6 +37,9 @@ export function createFetchCookiecutterAction(options: {
     url: string;
     targetPath?: string;
     values: JsonObject;
+    copyWithoutRender?: string[];
+    extensions?: string[];
+    imageName?: string;
```

Comment:
@Rugvip are we happy with `imageName`?

Reply:
Yep and then if we wanna be able to globally configure a different default image we can add that later